mandatory/4-tax.js

```diff
@@ -17,7 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(currency) {
+let first=calculateSalesTax(currency);
```

Comment:
Think about your indenting here - how can you update these variable declarations to make your code more readable. Also think about the naming of your variables here again. For example, what does the variable `first` mean ?